src/Parser/Monk/Brewmaster/Modules/Normalizers/GarothiWorldbreakerMelee.js

```diff
@@ -0,0 +1,37 @@
+import EventsNormalizer from 'Parser/Core/EventsNormalizer';
+import HIT_TYPES from 'Parser/Core/HIT_TYPES';
+import SPELLS from 'common/SPELLS';
+
+const GAROTHI_WORLDBREAKER = 2076;
```

Comment:
Arguably you could import https://github.com/WoWAnalyzer/WoWAnalyzer/blob/9e55fec255bac851822c2ce9636429ad2d963399/src/Raids/AntorusTheBurningThrone/GarothiWorldbreaker.js instead of making a new const, but as this number will never change I'm not sure it will ever matter.

Reply:
Missed that module, I'll import just to keep everything in one place.